cardano_node_tests/tests/test_transactions.py

```diff
@@ -1588,6 +1590,103 @@ def test_out_of_bounds_amount(
             exc_val = str(exc)
             assert "out of bounds" in exc_val or "exceeds the max bound" in exc_val
 
+    @allure.link(helpers.get_vcs_link())
+    @pytest.mark.skipif(not common.BUILD_USABLE, reason=common.BUILD_SKIP_MSG)
+    @hypothesis.given(amount=st.integers(max_value=MIN_LOVELACE_AMOUNT))
+    @common.hypothesis_settings()
+    def test_build_transfer_amount_bellow_minimum(
+        self,
+        cluster: clusterlib.ClusterLib,
+        payment_addrs: List[clusterlib.AddressRecord],
+        pbt_highest_utxo: clusterlib.UTXOData,
+        amount: int,
+    ):
+        """Try to build a transaction with amount bellow the minimum lovelace required.
+
+        Uses `cardano-cli transaction build` command for building the transactions.
+
+        Expect failure.
+        """
+        temp_template = f"{common.get_test_id(cluster)}_{amount}"
+
+        src_address = payment_addrs[0].address
+        dst_address = payment_addrs[1].address
+
+        with pytest.raises(clusterlib.CLIError) as excinfo:
+            cluster.cli(
+                [
+                    "transaction",
+                    "build",
+                    "--testnet-magic",
+                    str(cluster.network_magic),
+                    "--tx-in",
+                    f"{pbt_highest_utxo.utxo_hash}#{pbt_highest_utxo.utxo_ix}",
+                    "--change-address",
+                    src_address,
+                    "--tx-out",
+                    f"{dst_address}+{amount}",
+                    "--out-file",
+                    f"{temp_template}_tx.body",
+                ]
+            )
+        assert ("Minimum UTxO threshold not met for tx output" or "Negative quantity") in str(
+            excinfo.value
+        )
+
+    @allure.link(helpers.get_vcs_link())
+    @hypothesis.given(amount=st.integers(max_value=MIN_LOVELACE_AMOUNT))
+    @common.hypothesis_settings()
+    def test_transfer_amount_bellow_minimum(
+        self,
+        cluster: clusterlib.ClusterLib,
+        payment_addrs: List[clusterlib.AddressRecord],
+        pbt_highest_utxo: clusterlib.UTXOData,
+        amount: int,
+    ):
+        """Try to build a transaction with amount bellow the minimum lovelace required.
+
+        Uses `cardano-cli transaction build-raw` command for building the transactions.
+
+        Expect failure.
+        """
+        temp_template = f"{common.get_test_id(cluster)}_{amount}"
+
+        src_address = payment_addrs[0].address
+        dst_address = payment_addrs[1].address
+
+        fee = 200_000
+
+        src_init_balance = cluster.get_address_balance(src_address)
+        out_file = f"{temp_template}.body"
+
+        with pytest.raises(clusterlib.CLIError) as excinfo:
```

Comment:
Use this only for the function that actually fails, that is `cluster.submit_tx`.

Reply:
I used this approach because if we try to build the transaction with negative numbers will fails in that step, but if the amount is >= 0 will only fail on submit